### DISCUSSION TITLE
What is the difference between STC and LTC ?

### DISCUSSION THREAD

**Discussion by ghost** [official-stockfish/Stockfish#4189] [Q&A] [answered]
Hi, how to use Stock FishTest to find that  commits is good or not ?

**Comment by atumanian** [comment, marked as answer]
Hi! Here's a [Fishtest wiki page](https://github.com/glinscott/fishtest/wiki) to get started.

**Reply by ghost** [reply]
Thanks @atumanian

**Reply by atumanian** [reply]
> Thanks @atumanian

You're welcome. More specifically, you can start fron this page: [Creating my first test](https://github.com/glinscott/fishtest/wiki/Creating-my-first-test)
